src/Microsoft.SqlTools.ServiceLayer/Connection/Contracts/GetConnectionStringParams.cs

```diff
@@ -15,11 +15,16 @@ public class GetConnectionStringParams
         /// </summary>
         public string OwnerUri { get; set; }
 
+        /// <summary>
+        /// Connection information of the connection
+        /// </summary>
+        public ConnectionDetails ConnectionDetails { get; set; }
+
         /// <summary>
         /// Indicates whether the password should be return in the connection string
         /// default is set to false
         /// </summary>
-        public bool IncludePassword { get; set; }
+        public bool? IncludePassword { get; set; }
```

Comment:
I am wondering why we need to make this nullable?

Reply:
Yeah, why did you make this change @VasuBhog? As far as I can tell the ADS side always specifies this when sending the message so I'm not sure where we'd expect this to ever be undefined.

Reply:
Switched it back.